Formula/aubio.rb

```diff
@@ -25,20 +25,38 @@ class Aubio < Formula
   depends_on "numpy"
   depends_on "python@3.9"
 
+  # Replace python reference with python3
+  patch :DATA
```

Comment:
Maybe we can just do
```ruby
ENV.prepend_path "PATH", Formula["python@3.9"].opt_libexec/"bin"
```
?

Reply:
I wanted to do that too but it seems the formula doesn't have a `bin/python`:

```
$ ls $(brew --prefix python@3.9)/bin
2to3  2to3-3.9  idle3  idle3.9  pip3  pip3.9  pydoc3  pydoc3.9  python3  python3-config  python3.9  python3.9-config  wheel3
```

Reply:
I'm not entirely sure if that will work as it seems that a particular script has a hard coded reference to a `python` executable that doesn't exist in the `python@3.9` bottle.

Reply:
It does in `libexec`:
```
❯ ls $(brew --prefix python@3.9)/libexec/bin
idle  pip  pydoc  python  python-config  wheel
```

Reply:
ah, perfect!